resources/views/cart/partials/product/price.blade.php

```diff
@@ -7,7 +7,9 @@
     </div>
 </td>
 <td class="flex items-center font-medium max-md:w-1/3 md:table-cell *:mx-auto">
-    <x-rapidez-ct::input.quantity/>
+    <template v-if="item.is_available">
+        <x-rapidez-ct::input.quantity/>
```

Comment:
Out of scope but we can use the quantity select from the rapidez core?

Reply:
Just like the other feedback, its out of scope and we have issues ready for this subject